### ISSUE TITLE
Session Management Error

### ISSUE THREAD

**Issue by FrankXinqi** [open-mmlab/mmdetection#1611]
Hi, I try to run the first example, as
`python tools/test.py configs/faster_rcnn_r50_fpn_1x.py checkpoints/faster_rcnn_r50_fpn_1x_20181010-3d1b3351.pth --show`
But, get an session management error, as
`loading annotations into memory...`
`Done (t=0.60s)`
`creating index...`
`index created!`
`[                              ] 0/5000, elapsed: 0s, ETA:Qt: Session management error: None of the authentication protocols specified are supported`

**Comment by hellock** [comment]
You need to figure out if your opencv is installed with the support of QT GUI.

**Comment by FrankXinqi** [comment]
Hi Thanks. But, I still could not figure this out.
I have tried to install `mmdection` on another machine. It also comes to this problem in the end...
I install pyqt and opencv through conda install. It does not work.
Do you mean that I need to compile an opencv from scratch with qt? Could you hand on more hints?

**Comment by FrankXinqi** [comment]
Hi @hellock. Thanks for your help. I think I have solved the problem.
First of all, the `INSTALL guide of mmdection` does not directly work for me. So, I have to install `mmcv first`.
Then, I have found that for my new conda environment, `opencv-python` is installed when I execute `'pip install mmcv'`. But, this pip install still does not work for me.
My solution is to install mmcv from source using the [mmcv link](https://github.com/open-mmlab/mmcv), but do not associated with -e.
`git clone https://github.com/open-mmlab/mmcv.git`
`cd mmcv`
`pip install .`
This process also installs other dependencies, in which a correct version of `opencv-python` is in.

I guess the install `mmcv` from pip install gives us an old version of mmcv, which specifies a different dependency of `opencv` than we need now.
